Resource/template/admin/product_rank.twig

```diff
@@ -19,18 +19,18 @@ You should have received a copy of the GNU General Public License
 along with this program; if not, write to the Free Software
 Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
 #}
-{% extends 'default_frame.twig' %}
+{% extends '@admin/default_frame.twig' %}
 
 {% set menus = ['product', 'class_category'] %}
 
 {% block title %}商品管理{% endblock %}
 {% block sub_title %}商品並び替え{% endblock %}

```

Comment:
Please change to message translator

Reply:
@lqdung-lockon @toannguyen-lockon @hoand-vn done